docs/challenge-000.md

```diff
@@ -1,9 +1,51 @@
-# Challenge 0: Set up your development environment with Dev Containers
+# Challenge 0: Set up your development environment within a development container
 
 ## Challenge Objective
+The goal for this challenge is to set up and verify your dev environment in a VS Code dev container.
 
 ## Brief Guidance
 
+### Install Prerequisites
+Ensure you have a Microsoft Windows 10 or MacOS powered workstation with the following installed:
+- [Docker Desktop for Windows/MacOS](https://www.docker.com/products/docker-desktop)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Visual Studio Code - Remote Development Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+- [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/)
+- [Storage emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator) (Windows) or [create a storage account in Azure](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) (Mac)
+- [Postman](https://www.postman.com/downloads/)
+
+### Set up a VS Code dev container
+
+We'll be using a [dev container](https://code.visualstudio.com/docs/remote/containers) for this OpenHack to seamlessly set up your environment. To get started, follow these steps:
+
+1. Install and configure Docker for your operating system.
+
+    Windows / macOS:
+    1. Install Docker Desktop for Windows/Mac.
+    2. If not using WSL2 on Windows, right-click on the Docker task bar item, select **Settings / Preferences** and update **Resources > File Sharing** with any locations your source code is kept. See [tips and tricks](https://code.visualstudio.com/docs/remote/troubleshooting#_container-tips) for troubleshooting.
+    3. Enabling [Windows WSL2 back-end](https://aka.ms/vscode-remote/containers/docker-wsl2): Right-click on the Docker taskbar item and select **Settings**. Check **Use the WSL2 based engine** and verify your distribution is enabled under **Resources > WSL Integration.**
+
+    Linux:
+
+    1. Follow the official install [instructions for Docker CE/EE for your distribution](https://docs.docker.com/install/#supported-platforms). If you are using Docker Compose, follow the [Docker Compose directions](https://docs.docker.com/compose/install/) as well.
+    2. Add your user to the `docker` group by using a terminal to run: `sudo usermod -aG docker $USER`
+    3. Sign out and back in again so your changes take effect.
+
+### Open the project in your dev container
+
+Next clone the repo locally and set up a dev container for your existing project on your filesystem.
```

Comment:
doesn't the dev container come down as part of the clone?? this whole section should be unnecessary and they should just have to "reopen in container" from the pallette.

Reply:
yes, you're right. i have simplified to remove the steps i think are unnecessary.